ResearchKit/Consent/ORKConsentLearnMoreViewController.h

```diff
@@ -38,6 +38,7 @@ NS_ASSUME_NONNULL_BEGIN
 @interface ORKConsentLearnMoreViewController : UIViewController
 
 - (instancetype)initWithHTMLContent:(NSString *)content;
+- (instancetype)initWithContentRequest:(NSURLRequest *)contentRequest;
```

Comment:
I don't think it's a good idea to take something that's not a file URL here... we have no loading indication or navigation controls in this view.